src/fastspell/fastspell.py

```diff
@@ -105,10 +110,36 @@ def search_hunspell_dict(self, lang_code):
                     logging.error("Failed building Hunspell object for " + lang_code)
                     logging.error("Aborting.")
                     exit(1)
-        else:
-            raise RuntimeError(f"It does not exist any valid dictionary directory"
-                               f"for {lang_code} in the paths {self.hunspell_paths}."
+
+        if hunspell_obj is None:
```

Comment:
This change is unrelated, I needed it to load as many dictionaries as possible from my system (as I was testing with Italian, and it is not available in fastspell-dictionaries).

Reply:
I guess this change could be useful in its own right, so users of fastspell can more easily load more dictionaries.

Reply:
Let me know if you want to land this change (after cleaning it up of course) and I'll open a PR for it.

Reply:
This was to allow loading a dictionary more easily, e.g. if "it" is the language and there's "it-IT.dic" in the system, it will assume that's the one to load (even though there might be others like "it_CH.dic").